apps/ima/views/immigration-adviser-details.html

```diff
@@ -0,0 +1,27 @@
+{{<partials-page}}
+  {{$page-content}}
+     <p>{{#t}}pages.immigration-adviser-details.paragraph{{/t}}</p>
+    {{#input-text}}legal-representative-fullname{{/input-text}}
+
+    {{#input-text}}legal-representative-organisation{{/input-text}}
+
+    <span class="govuk-label">
+      <strong>{{#t}}pages.immigration-adviser-details.address{{/t}}</strong>
+    </span>
+    {{#input-text}}legal-representative-house-number{{/input-text}}
+    {{#input-text}}legal-representative-street{{/input-text}}
+    {{#input-text}}legal-representative-townOrCity{{/input-text}}
+    {{#input-text}}legal-representative-county{{/input-text}}
+    {{#input-text}}legal-representative-postcode{{/input-text}}<br>
+    {{#input-text}}legal-representative-phone-number{{/input-text}}
+    <span class="govuk-label">
+      <strong>{{#t}}pages.is-legal-representative-email{{/t}}</strong>
+    </span>
+    {{#renderField}}is-legal-representative-email{{/renderField}}
+   
+    <span class="govuk-label other-details">
+      {{#t}}Answer all the following questions as if you are the person on the duty to remove notice{{/t}}
```

Comment:
According to the latest Figma designs, it should be 'Answer all the following questions as if you are the person on the Illegal Migration Act notice.'